exercises/palindrome-products/.meta/src/reference/java/PalindromeCalculator.java

```diff
@@ -3,6 +3,8 @@
 public class PalindromeCalculator {
 
     public SortedMap<Long, List<List<Integer>>> getPalindromeProductsWithFactors(int minFactor, int maxFactor) {
+    	if(minFactor > maxFactor) throw new IllegalArgumentException("invalid input: min is " + minFactor + " and max is " + maxFactor);
```

Comment:
This would read more nicely if it was presented as:

```
if (minFactor > maxFactor) {
    throw new IllegalArgumentException("invalid input: min is " + minFactor + " and max is " + maxFactor);
}
```